#include <bachelor/Topic.h>

std::map<Topic, std::string> NodeName = 
{
    {ImHere_CamSim,  "CameraSimulator_Node"},
    {ImHere_ECU,     "ECU_Node"},
    {ImHere_Visual,  "Visualizer_Node"},
    {ImHere_LaneDet, "LaneDetector_Node"},
    {ImHere_LimDet,  "LimitDetector_Node"},
    {ImHere_StopDet, "StopDetector_Node"}
};